reaktive/src/commonMain/kotlin/com/badoo/reaktive/base/DisposableEmitter.kt

```diff
@@ -3,6 +3,7 @@ package com.badoo.reaktive.base
 import com.badoo.reaktive.disposable.Disposable
 import com.badoo.reaktive.disposable.DisposableWrapper
 
+@PublishedApi
```

Comment:
Not sure about this. This annotation makes the class effectively public and it will be subject for binary compatibility. Maybe just remove?